testing/python/approx.py

```diff
@@ -277,13 +289,13 @@ def test_error_messages_with_different_verbosity(self, assert_approx_raises_rege
             a,
             b,
             [
+                r"  ",
                 r"  comparison failed. Mismatched elements: 20 / 20:",
                 rf"  Max absolute difference: {SOME_FLOAT}",
                 rf"  Max relative difference: {SOME_FLOAT}",
                 r"  Index \| Obtained\s+\| Expected",
                 rf"  \(0,\)\s+\| {SOME_FLOAT} \| {SOME_FLOAT} ± {SOME_FLOAT}",
                 rf"  \(1,\)\s+\| {SOME_FLOAT} \| {SOME_FLOAT} ± {SOME_FLOAT}",
-                rf"  \(2,\)\s+\| {SOME_FLOAT} \| {SOME_FLOAT} ± {SOME_FLOAT}...",
```

Comment:
I assume the `...` here is added by pytest to indicate truncation. If so, shouldn't it be now on the previous line?

Reply:
Just to make sure, these are literal dots, and the assert is a regex, right? If so, it would be nice to escape the `.`s to make it clear and not overmatch: `\.\.\.`. And maybe also add anchors `^` and `$`.

Reply:
The regex is good :+1: